lib/variations.js

```diff
@@ -4,16 +4,16 @@ const simplePrefs = require('sdk/simple-prefs');
 let Advise = null;
 
 const variations = {
-  'passive-panel'() {
+  'regular-sync-offer'() {
     Advise = new Advisor();
     Advise.start();
   },
-  'auto-open-panel'() {
-    simplePrefs.prefs.autoOpenPanel = true;
+  'new-features-sync-offer'() {
```

Comment:
With some testing I found that the code is run every time the add-on starts up. I'll pass the branch name in as an argument to the advisor class.